### PR TITLE
Added initial support for Java and RHEA QE clients

### PR DESCRIPTION
**Pull request by fgiorgetti**

### CODE REVIEW

File: pkg/api/client/amqp/qeclients/argparser/java/parser_test.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package java
+
+import (
+	"reflect"
+	"testing"
+)
+
+// TestUrl validates the Url function for the cli-java parser
+func TestUrl(t *testing.T) {
+	args := Url("amqp://user:pass@127.0.0.1:5672/my/address/name")
```
Comment (nicob87) [inline pkg/api/client/amqp/qeclients/argparser/java/parser_test.go]: what is expected to happen if you call it with corrupted url? maybe the test can  also call the function with malformed, and other kinds of incorrect urls.

Comment (fgiorgetti) [reply pkg/api/client/amqp/qeclients/argparser/java/parser_test.go]: https://github.com/rh-messaging/shipshape/pull/

Comment (nicob87) [reply pkg/api/client/amqp/qeclients/argparser/java/parser_test.go]: great!

File: pkg/framework/pod.go [hunk 2]
```diff
@@ -112,6 +112,18 @@ func (cb *ContainerBuilder) AddArgs(args ...string) *ContainerBuilder {
 	return cb
 }
 
+// EnvVar appends a given list of environment variables into the container
```
Comment (nicob87) [inline pkg/framework/pod.go]: "appends a given list" or just a single environment variable?

**Comment by nicob87** [comment]
Looks good to me!

**Pull request merged**